### PR TITLE
Use smaller symbols for pages_left_book in compact_items

### PR DESCRIPTION
**Pull request by JasonInOttawa**
Use the (smaller) "‹" or "›" symbols in pages_left_book, to match pages_left, in compact_items

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7828)

### CODE REVIEW

**Comment by NiLuJe** [comment]
They're the same though, which is possibly confusing, and why I went with something else ;).

**Comment by JasonInOttawa** [comment]
I prefer the sameness :) I defer to popular opinion though.

The pages left in book is differentiated by the "/<n>" after.

**Comment by NiLuJe** [comment]
> The pages left in book is differentiated by the "/n" after.

Oh, right, true enough ;).

**Comment by JasonInOttawa** [comment]
This is what things look like with the larger icon:

Screen Shot 2021-06-09 at 4 13 37 PM

This is what they look like with the smaller icon:

Screen Shot 2021-06-09 at 4 13 14 PM

Smaller looks tidier to me. (I like having the progress bar as long as possible - it makes the chapter markers easier to make out. This book only has two chapters in the TOC, though.)

**Comment by JasonInOttawa** [comment]
Sorry for the closing/reopening and the extra build. I hit the wrong button.

**Comment by JasonInOttawa** [comment]
Any chance we could pack this into 2021.06? Do folks prefer the larger symbol?

**Comment by pazos** [comment]
> Any chance we could pack this into 2021.06? Do folks prefer the larger symbol?

I do prefer the larger symbols, but as long as it just affect "compact" symbols I'm fine with it.

**Comment by JasonInOttawa** [comment]
This PR only changes the symbols in compact_items

**Comment by JasonInOttawa** [comment]
Is this something I need to request a review for? I don't see a "request review" button anywhere. 

Also aware that this is crazy-low priority :)

**Comment by jimman2003** [comment]
that button is beside the reviewers sections besides the reviewer's name

**Comment by NiLuJe** [comment]
I mean, I haven't much more to say: *I* wouldn't use identical symbols, on principle ;p.

Then again:

* I don't use the feature
* And it's true that there are other bits of the widgets that help tell 'em apart.

TL;DR: Not a fan, but why not. (Then again, I already wasn't a fan of the feature itself, so ^^).

**Comment by Frenzie** [comment]
Merging then. I have no particular opinion on this `pages_left_book` thing, although I agree that using exactly the same seems potentially slightly confusing.

**Comment by JasonInOttawa** [comment]
I guess this one is fairly unique in that it uses two symbols: the < sign and the / symbol. Closest thing I can see is "pages left in book" vs "current page in chapter" which is / vs //

I agree if pages left in book did not have the "/n" bit that it could be confusing.
